main.tf

```diff
@@ -10,4 +10,11 @@ resource "aws_lb" "test" {
   tags = {
     Environment = "production"
   }
+
+  access_logs {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure access logging is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS classic load balancers do not have access logging enabled.Access logging should be enabled in order to analyze statistics, diagnose issues, and retain data for regulatory or legal purposes.
 
**Remediation**
To enable access logs for your load balancer, you must specify the name of the Amazon S3 bucket where the load balancer will store the logs. You must also attach a bucket policy to this bucket that grants Elastic Load Balancing permission to write to the bucket.

source: accurics-stagetwo